gulpfile.js

```diff
@@ -149,6 +149,7 @@ gulp.task('watch', ['webpack-watch'], () => {
   gulp.watch([
     `${paths.ICONS_DIR}**/*.svg`,
     `${paths.SRC_TEMPLATES_DIR}_icon_template.lodash`,
+    '!content/static/assets/*.css',
```

Comment:
What is this intended to do? It already shouldn't be watching this path, and also I think `!` patterns have to be last in the list for this to do anything.